plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -649,12 +664,27 @@ private Document getTableMetadata(String schemaName, String tableName)
 
     public boolean collectionExists(MongoDatabase db, String collectionName)
     {
+        //Use when case-insensitive matching
+        List<String> candidates = new ArrayList<>();
```

Comment:
Rename to `remoteSchemaTableNames` or `collectionNames`.

Reply:
I've renamed it to `remoteSchemaTableNames`